comp/core/workloadmeta/collectors/util/image_metadata_util_test.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2023-present Datadog, Inc.
 
-// Package util contains utility functions for image metadata collection
+// Package util contains utility functions for workload metadata collectors
```

Comment:
I didn't realize this because not all the files in the `utils` directory appeared in the diff before, but it's not needed to write the same documentation on every file. We can just document one or create a `doc.go` file. I personally prefer the second option if there isn't a clearly "main" file or entry point inside the package, like in this case where there are 3 with utils for different objects.